trendico/urls.py

```diff
@@ -6,9 +6,13 @@
     path('get_products/<str:category>/',
          views.ProductsByCategoryView.as_view(), name='product-category'),
     path('store/<str:name>', views.StoreView.as_view(), name='store'),
-    path('product/', views.product, name='product'),
+    path('product/<str:name>', views.ProductView.as_view(), name='product'),
     path('checkout/', views.checkout, name='checkout'),
     path('login/', views.LoginView.as_view(), name='login'),
     path('logout/', views.LogoutView.as_view(), name="logout"),
     path('register/', views.RegisterView.as_view(), name="register"),
+    path('add_to_cart/', views.AddToCartView.as_view(), name="add_to_cart"),
```

Comment:
AddToCartView and CartSummaryView
can be handled in only one class e.g CartView which will have get() and post() function
so when you want to get summary, you will call this /cart with GET and for adding something to cart, call this /cart as a POST